docs/adguard-browser-extension/solving-problems/update-rules.md

```diff
@@ -0,0 +1,94 @@
+---
+title: How to update declarative rules
+sidebar_position: 2
+---
+
```

Comment:
We should mention that in most of the cases using "User rules" in the extension should suffice. However, in some problematic cases you may want to modify rules that are bundled with the extension statically.